### PR TITLE
Modified code to allow for images

### PR DESCRIPTION
**Pull request by stanleynwakamma**
Modified the doPost() function in the MessageServlet.java class to include image capabilities.

Modified the user-page.css file to restrict the image size

Updated Datastore index by adding a recipient field to the index.yaml file

### CODE REVIEW

File: src/main/java/com/google/codeu/servlets/MessageServlet.java [hunk 1]
```diff
@@ -76,10 +76,15 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     }
 
     String user = userService.getCurrentUser().getEmail();
-    String text = Jsoup.clean(request.getParameter("text"), Whitelist.basic());
+
+    String userText = Jsoup.clean(request.getParameter("text"), Whitelist.none());
```
Comment (kyrabusser) [inline src/main/java/com/google/codeu/servlets/MessageServlet.java]: Does this still allow @hengelman's work to display styled text work?

File: src/main/webapp/WEB-INF/index.yaml [hunk 2]
```diff
@@ -16,5 +16,6 @@ indexes:
 - kind: Message
   properties:
   - name: user
+  - name: recipient
```
Comment (kyrabusser) [inline src/main/webapp/WEB-INF/index.yaml:19]: Is this related to images?

**Review by kyrabusser — COMMENTED** [review]
Nice! 🎉 🖼

Do you have a screenshot of what these images look like?

**Comment by stanleynwakamma** [comment]
Yes sure. A screenshot is attached below
image capture

File: src/main/webapp/WEB-INF/index.yaml (hunk 2 above)
Comment (stanleynwakamma) [reply src/main/webapp/WEB-INF/index.yaml:19]: Not really, but that was a modification to the familiarization projects (Step 7: Update the Datastore index) which was recently added

File: src/main/java/com/google/codeu/servlets/MessageServlet.java (hunk 1 above)
Comment (stanleynwakamma) [reply src/main/java/com/google/codeu/servlets/MessageServlet.java]: For some reasons, the changes she made to the code isn't reflecting in my system

**Comment by stanleynwakamma** [comment]
@kyrabusser How do I check that Hannah's features still work? But I don't think I broke her code though 😆

**Comment by kyrabusser** [comment]
Sounds like a good question for @hengelman ! Hannah, what should Stanley do to verify styled text?

File: src/main/webapp/css/user-page.css [hunk 3]
```diff
@@ -36,12 +36,14 @@
 
 /* Give the message body a margin so the text
    isn't too close to the border. */
-.message-body {
-  margin: 5px;
```
Comment (kyrabusser) [inline src/main/webapp/css/user-page.css]: Why delete this margin?

File: src/main/java/com/google/codeu/servlets/MessageServlet.java (hunk 1 above)
Comment (hengelman) [reply src/main/java/com/google/codeu/servlets/MessageServlet.java]: This won't work with the styled text it needs to be .basic or all HTML will be disallowed

**Comment by hengelman** [comment]
@stanleynwakamma try inputting messages with any of the modifications allowed in whitelist.basic such as bold, underline, etc. Also I'm not sure but you might need to change it to .basicWithImages() to allow images but adding images works without that its probably fine

**Comment by stanleynwakamma** [comment]
image capture 2

**Comment by stanleynwakamma** [comment]
@kyrabusser @hengelman  I think everything works well. Please see my attachment above. thanks

File: src/main/webapp/css/user-page.css (hunk 3 above)
Comment (stanleynwakamma) [reply src/main/webapp/css/user-page.css]: I thought it does the same as .message-body img {

File: src/main/java/com/google/codeu/servlets/MessageServlet.java (hunk 1 above)
Comment (stanleynwakamma) [reply src/main/java/com/google/codeu/servlets/MessageServlet.java]: Yea I changed it back to basic in my last commit 
dd19931

File: src/main/webapp/css/user-page.css (hunk 3 above)
Comment (kyrabusser) [reply src/main/webapp/css/user-page.css]: Not exactly--this adds a margin to every message, because it selects everything with the .message-body css class. The .message-body img only styles image tags, so it doesn't change just plain messages.

If you look at before this change, there's a little bit of space between the border and the message, but with this deleted the words are right up against the edges.

**Comment by stanleynwakamma** [comment]
@kyrabusser Please checkout commit 38696cd  Just fixed it. Thanks 😃

**Review by kyrabusser — APPROVED** [review]
Nice work! :tada:

**Comment by kyrabusser** [comment]
(Also, I love your example images! So cute! :smile: )
